src/regenerate.rs

```diff
@@ -225,7 +225,7 @@ fn save_remainings(infos: &HashMap<String, Vec<(Option<TypeStruct>, Vec<String>)
         }
     }
     if remainings < 1 {
-        remove_file(OUTPUT_COMMENT_FILE);
+        let _ = remove_file(OUTPUT_COMMENT_FILE);
```

Comment:
In these cases, you could alternatively call "unwrap" method on it. However, I really need to put in place a full error strategy.